content/css/concepts/selectors/selectors.md

```diff
@@ -27,43 +48,52 @@ p {
 }
 ```
 
-## Id Selector
+## Selecting by Attribute
 
-The `id` selector uses the `id` attribute of an HTML element to select a specific element.
+Most commonly, the `class` and `id` attributes are assigned to elements for styling purposes. An `id` cannot start with a number.
 
-The `id` of an element is unique within a page, so the `id` selector is used to select one unique element!
+### Syntax
 
-To select an element with a specific `id`, write a hash (`#`) character, followed by the `id` of the element.
+```css
+#id-name {
+  /* declarations here */
+}
 
-The CSS rule below will be applied to the HTML element with `id="para1"`:
+.class-name {
+  /* declarations here */
+}
 
-```css
-#para1 {
-  text-align: center;
-  color: red;
+element[attribute='value'] {
+  /* declarations here */
 }
 ```
 
-**Note:** An `id` name cannot start with a number!
-
-## Class Selector
+The `id` of an element is unique within a page, so the `id` selector is used to select one unique element. An element can be selected with a specific `id`, by using a hash (`#`) character, followed by the `id` of the element.
 
-The class selector selects HTML elements with a specific class attribute.
+Conversely, multiple `class` values can be assigned to a single element. Elements are selected by `class` with a period (`.`) character, followed by the class name.
```

Comment:
"Conversely," doesn't work here since this is not mirroring the prior statement. It's talking about multiple class values in an element, not on a page. This also should mention that class values, unlike id values, can be duplicated on a page with different elements.